packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -729,3 +730,67 @@ test(
   'metadataObjectFromCMRFile throws PreconditionFailed when ETag does not match CMR UMMG JSON metadata file',
   testMetadataObjectFromCMRFile('s3://bucket/fake.cmr.json')
 );
+
+test.serial('publish2CMR passes cmrRevisionId to publishECHO10XML2CMR', async (t) => {
+  const cmrFileObject = { filename: 'test.cmr.xml', granuleId: 'testGranuleId' };
+  const updatedXmlFile = { ...cmrFileObject, metadataObject: {} };
+
+  const publishECHO10XML2CMRSpy = sinon.spy(() => Promise.resolve());
+  const revertPublishECHO10XML2CMRSpy = cmrUtil.__set__('publishECHO10XML2CMR', publishECHO10XML2CMRSpy);
+  const credentials = {};
+  const cmrRevisionId = Math.floor(Math.random() * 100);
+
+  t.teardown(() => {
+    revertPublishECHO10XML2CMRSpy();
+  });
+  await cmrUtil.publish2CMR(updatedXmlFile, credentials, cmrRevisionId);
+  t.is(publishECHO10XML2CMRSpy.getCall(0).args[2], cmrRevisionId);
+});
+
+test.serial('publish2CMR passes cmrRevisionId to publishUMMGJSON2CMR', async (t) => {
+  const cmrFileObject = { filename: 'test.cmr.json', granuleId: 'testGranuleId', metadataObject: {} };
+  const cmrRevisionId = Math.floor(Math.random() * 100);
+  const credentials = {};
+
+  const publishUMMGJSON2CMRSpy = sinon.spy(() => Promise.resolve());
+  const revertPublishUMMGJSON2CMRSpy = cmrUtil.__set__('publishUMMGJSON2CMR', publishUMMGJSON2CMRSpy);
+
+  t.teardown(() => {
+    revertPublishUMMGJSON2CMRSpy();
+  });
+
+  await cmrUtil.publish2CMR(cmrFileObject, credentials, cmrRevisionId);
+  t.is(publishUMMGJSON2CMRSpy.getCall(0).args[2], cmrRevisionId);
+});
+
+test.serial('publishECHO10XML2CMR passes cmrRevisionId to ingestGranule', async (t) => {
+  const cmrFileObject = { filename: 'test.cmr.xml', granuleId: 'testGranuleId', metadataObject: {} };
+  const conceptId = randomString();
+  const credentials = {};
+  const cmrRevisionId = Math.floor(Math.random() * 100);
+
+  const ingestGranuleSpy = sinon.stub(CMR.prototype, 'ingestGranule').returns({ result: { 'concept-id': conceptId, 'revision-id': cmrRevisionId } });
```

Comment:
Nit: I'm wondering if we should bother including `revision-id` in our mock here. Even though CMR does return that value (is this true?), we're not really relying on that in our code anymore, so I'm hesitant to mock it because it seems like we do need it